### ISSUE TITLE
Exception shown on listing new show resulting in societies not being added to the show

### ISSUE THREAD

**Issue by GKFX** [camdram/camdram#648]
https://sentry.io/organizations/camdram-webteam/issues/1003710895/?project=1273126&referrer=github_plugin

```
InvalidArgumentException: Field "slug" is not a valid field of the entity "Acts\CamdramBundle\Entity\Show" in PreUpdateEventArgs.
  File "src/Acts/CamdramBundle/EventListener/ShowListener.php", line 165, in manageSlugChange
    $newSlug = $slugRepo->findOneBySlug($event->getNewValue("slug"));
  File "src/Acts/CamdramBundle/EventListener/ShowListener.php", line 68, in preUpdate
    $this->manageSlugChange($show, $event);
  File "src/Acts/CamdramBundle/Controller/AbstractRestController.php", line 145, in postAction
    $em->flush();
  File "src/Acts/CamdramBundle/Controller/ShowController.php", line 114, in postAction
    return parent::postAction($request);
  File "web/app.php", line 28
    $response = $kernel->handle($request);
...
(16 additional frame(s) were not displayed)
```

**Comment by GKFX** [comment]
Closing on the assumption that this is some sort of artefact of the release process. I can't reproduce this, it hasn't happened again, and reading both our and Doctrine's source code suggests that it's logically impossible for this to happen. Specifically, https://github.com/camdram/camdram/blob/d490663ea9f1f661fa7f84ebd48206aad77a9f89/src/Acts/CamdramBundle/EventListener/ShowListener.php#L67
checks that the slug field is present in an array and
https://github.com/camdram/camdram/blob/d490663ea9f1f661fa7f84ebd48206aad77a9f89/src/Acts/CamdramBundle/EventListener/ShowListener.php#L165
retrieves the value.

**Comment by GKFX** [comment]
Clearly this is a) possible and b) not related to the release in any way as it has recurred. Only occurs when listing shows which should be approved automatically as the user is an admin of the society, doesn't occur if a site admin lists a show with a society, but does happen if a site admin edits one of the affected shows and tries to add the missing society back in.

**Comment by GKFX** [comment]
Fixed in 923655df33d0bfebf08c245e5cae82d6a0c74bf3.
